src/Repository/SponsorCategoryRepository.php

```diff
@@ -47,6 +47,18 @@ public function remove(SponsorCategory $entity, bool $flush = true): void
         }
     }
 
+    public function findAllWithVisibleSponsors(): array {
+        return $this->createQueryBuilder('sc')
+            ->addSelect('s')
+            ->leftJoin('sc.sponsors', 's')
+            ->andWhere("s.isVisible = :isVisible")
+            ->orWhere("s.isVisible is null")
```

Comment:
warum is null?

Reply:
Wenn du leere Categorien anzeigen willst. Dhaer auch der LeftJoin